documentation/asciidoc/accessories/display/display_intro.adoc

```diff
@@ -132,6 +132,27 @@ Then, disable automatic display detection by removing the following line from `c
 display_auto_detect=1
 ----
 
+[NOTE]
+====
+In Raspberry Pi OS _Bookworm_, a bug in https://github.com/WayfireWM/wayfire[Wayfire] currently prevents cursor movement rotation. There is no workaround for this bug in Wayfire. Instead, you can use https://github.com/labwc/labwc[`labwc`] (currently in beta).
```

Comment:
It incorrectly rotates the cursor when using WayVNC. Locally it is fine.

AIUI we aren't be looking for a fix in Wayfire.
Labwc is the route we are taking forward as it should be work reasonably on the earlier Pis, and hope to adopt it very soon.

Reply:
Yeah I agree but what I am saying is that the problem of the inverted mouse in wayvnc is a wayfire problem of not assigning something correctly. I am just recommending labwc to people that just want to make wayvnc with rotation work.

Reply:
Fixing this in a separate PR. Thanks!